web/modules/custom/dpl_event/dpl_event.module

```diff
@@ -322,10 +323,18 @@ function dpl_event_form_alter(array &$form, FormStateInterface $form_state, stri
     'eventinstance_default_edit_form',
     'eventinstance_default_add_form',
   ];
+
   if (!in_array($form_id, $forms)) {
     return;
   }
 
+  // Move the "Published" checkbox below "Create new revision".
+  if ($form_id === 'eventseries_default_edit_form' || $form_id === 'eventinstance_default_edit_form') {
```

Comment:
An idea, but not a requirement - do this, the same way as we do above:

```
if (in_array($form_id, ['eventseries_default_edit_form', 'eventinstance_default_edit_form']) {
```

or, a bit less verbose:

```
if (str_ends_with($form_id, '_edit_form')) {